### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by luwes** [julianburr/sublime-import-cost#5]
It hangs and sometimes crashes Sublime for files with a lot of imports.
Can it be executed async? Even when using cmd+p searching files it makes Sublime hang.

**Comment by julianburr** [comment]
Yeah, the performance is definitely a problem ... I used the async plugin hooks, but I think the node bridge might be synchronous atm. There is a minor improvement on the latest master, but still not satisfying. Will definitely try to find a solution for this when I find the time 👍

**Comment by dortamiguel** [comment]
I want to trigger manually import cost instead of by default. I  don't need to check it all the time, just after installing one package or reviewing some code.
